crypto/ring-signature/signer/src/traits.rs

```diff
@@ -116,15 +116,15 @@ impl<S: RingSigner> RingSigner for &S {
         signable_ring: &SignableInputRing,
         output_blinding: Scalar,
         rng: &mut dyn CryptoRngCore,
-    ) -> Result<RingMLSAG, Error> {
+    ) -> Result<RingMLSAG, SignerError> {
         (*self).sign(message, signable_ring, output_blinding, rng)
     }
 }
 
 /// An error that can occur when using an abstract RingSigner
 #[derive(Clone, Debug, Display, Eq, Hash, Ord, PartialEq, PartialOrd)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-pub enum Error {
+pub enum SignerError {
```

Comment:
Seems like this change is purely cosmetic, is there a reason not to keep it as just `Error`?

Reply:
it gets renamed to `SignerError` basically everywhere it's used, figured it makes sense to rename it at the root and avoid the collision everywhere else.

Reply:
Can we undo this change, then?